terraform/azure/iglu_server/terraform.tfvars

```diff
@@ -0,0 +1,46 @@
+# Will be prefixed to all resource names
+# Use this to easily identify the resources created and provide entropy for subsequent environments
+prefix = "iglu"
```

Comment:
We typically use `sp` as the prefix for everything... I think especially for the pipeline, just `pipeline` will be confusing as a prefix as it’s a generic word. (Not that `sp` is great... `snowplow` could be a good middle ground.)

Reply:
Agreed on this point - happy for it to be `snowplow` instead but it should probably end up being the same for all three modules.